basic_samples/SDS/Python/SDSPy/readme.rst

```diff
@@ -379,51 +378,51 @@ The following shows how this is done in the code:
 The process consists of two steps. First, the Property Override must be created, then the
 stream must be updated. Note that the sample retrieves three data points
 before and after updating the stream to show that it has changed. See
-the `Sds documentation <https://cloud.osisoft.com/documentation>`__ for
-more information about Sds Property Overrides.
+the `SDS documentation <https://ocs-docs.osisoft.com/Documentation/SequentialDataStore/Data_Store_and_SDS.html>`__ for
+more information about SDS Property Overrides.
 
-SdsViews
+SdsStreamViews
 -------
 
-A SdsView provides a way to map stream data requests from one data type 
-to another. You can apply an SdsView to any read or GET operation. SdsView 
+A SdsStreamView provides a way to map stream data requests from one data type 
+to another. You can apply an SdsStreamView to any read or GET operation. SdsStreamView 
 is used to specify the mapping between source and target types.
 
-Sds attempts to determine how to map Properties from the source to the 
+SDS attempts to determine how to map Properties from the source to the 
 destination. When the mapping is straightforward, such as when 
 the properties are in the same position and of the same data type, 
-or when the properties have the same name, Sds will map the properties automatically.
+or when the properties have the same name, SDS will map the properties automatically.
 
 .. code:: python
 
         rangeWaves = client.getRangeValues(namespaceId, stream.Id, WaveDataTarget, "1", 0, 3, False, SdsBoundaryType.ExactOrCalculated, automaticView.Id)
 
-To map a property that is beyond the ability of Sds to map on its own, 
-you should define an SdsViewProperty and add it to the SdsVeiw’s Properties collection.
+To map a property that is beyond the ability of SDS to map on its own, 
+you should define an SdsStreamViewProperty and add it to the SdsVeiw’s Properties collection.
 
 .. code:: python
 
-        vp2 = SdsViewProperty()
+        vp2 = SdsStreamViewProperty()
         vp2.SourceId = "Sin"
         vp2.TargetId = "SinInt"
         ...
-        manualView = SdsView()
+        manualView = SdsStreamView()
```

Comment:
I changed all View to StreamView in the code including variable names so this is the code now:
    manualStreamView = SdsStreamView()
    manualStreamView.Id = sampleStreamViewIntId
    manualStreamView.Name = "SampleIntStreamView"
    manualStreamView.TargetTypeId = waveIntegerType.Id
    manualStreamView.SourceTypeId = waveType.Id
    manualStreamView.Properties = [vp1, vp2, vp3, vp4]